examples/crhmc_cooling_gaussians/volume_example.cpp

```diff
@@ -0,0 +1,49 @@
+#include "generators/known_polytope_generators.h"
+#include "random_walks/random_walks.hpp"
+#include "volume/volume_cooling_gaussians_crhmc.hpp"
+
+#include <iostream>
+#include <fstream>
+#include "misc/misc.h"
+
+typedef double NT;
+typedef Cartesian<NT> Kernel;
+typedef typename Kernel::Point Point;
+typedef BoostRandomNumberGenerator<boost::mt11213b, double> RandomNumberGenerator;
+typedef HPolytope<Point> HPOLYTOPE;
+
+void calculateAndVerifyVolume(HPOLYTOPE& polytope) {
+    int walk_len = 100;
```

Comment:
does crhmc need such a large value for walk_length?

Reply:
I found better results with a higher value. Of course, it is not a necessity, but compared to something like 10, it is surely performing better (at least based on my observations and the other current parameters). I think we can later do a better analysis on how this is performing. @TolisChal was also suggesting some modifications for N and W within the gaussian annealing.

Reply:
In previous volume approximation methods best performance was obtained with walk_length=1.
However, we can check this later

Reply:
Regarding N and W they should be much smaller than their default values, e.g., N=1200 and W=300